### PR TITLE
Enable Chrome launcher in Karma

### LINKED ISSUE #986: Enable unit tests for Chrome (in Karma)
Might catch things like: https://github.com/aframevr/aframe/issues/983

### PR DESCRIPTION
**Pull request by donmccurdy**
This came up as part of my Controls refactor, thought I'd make a separate PR out of it.

Fixes #986.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,8 @@
     "release:push": "npm login && npm publish && git push --follow-tags",
     "start": "npm run dev",
     "test": "karma start ./tests/karma.conf.js",
+    "test:firefox": "karma start ./tests/karma.conf.js --browsers firefox_latest",
+    "test:chrome": "karma start ./tests/karma.conf.js --browsers Chrome",
```
Comment (donmccurdy) [inline package.json]: Eh, is this useful? I guess you can already just do:

```
npm run test -- --browsers Chrome
```

Comment (ngokevin) [inline package.json]: I think it's worth abstracting.

File: src/components/sound.js [hunk 2]
```diff
@@ -66,7 +66,12 @@ module.exports.Component = registerComponent('sound', {
 
   remove: function () {
     this.el.removeObject3D('sound');
-    this.sound.disconnect();
+    try {
```
Comment (dmarcos) [inline src/components/sound.js]: Is there a way to know if the sound is connected or not instead of doing a try/catch?

Comment (donmccurdy) [inline src/components/sound.js]: Nothing obvious in [THREE.Audio](https://github.com/mrdoob/three.js/blob/master/src/audio/Audio.js). Reaching into the object a bit, `this.sound.source.context.state === 'running'` looked promising but it's still throwing in that state. There's probably an event or something but i can't find much in the AudioNode spec. :neutral_face:

File: src/components/obj-model.js [hunk 3]
```diff
@@ -58,12 +58,14 @@ module.exports.Component = registerComponent('obj-model', {
...
     objLoader.load(objUrl, function (objModel) {
       // Apply material.
-      var material = el.components.material.material;
-      objModel.traverse(function (child) {
-        if (child instanceof THREE.Mesh) {
-          child.material = material;
-        }
-      });
+      var material = el.components.material;
+      if (material) {
```
Comment (ngokevin) [inline src/components/obj-model.js]: Do you think we have a bug? Material is a dependency so it should be initialized by now.

Comment (dmarcos) [inline src/components/obj-model.js]: Yeah. Checking for it should not be necessary

**Comment by ngokevin** [comment]
lgtm, we'd file a follow-up issue on that obj-model dep bug. can you squash?

File: src/components/obj-model.js (hunk 3 above)
Comment (ngokevin) [inline src/components/obj-model.js]: Filed it at https://github.com/aframevr/aframe/issues/1369 since this means it was an existing issue.

Comment (dmarcos) [inline src/components/obj-model.js]: Can we fix this in a separate PR?

Comment (ngokevin) [inline src/components/obj-model.js]: Yeah, I'm suggesting separate PR.

**Pull request merged**